lib/project_types/script/messages/messages.rb

```diff
@@ -76,6 +76,25 @@ module Messages
 
           packages_outdated_cause: "The following npm packages are out of date: %s.",
           packages_outdated_help: "Update them by running {{cyan:npm install --save-dev %s}}.",
+
+          invalid_build_script: "Invalid build script",
+          build_script_not_found: "Build script not found.",
+          build_script_suggestion: <<~HELP,
+            The root package.json should contain a script named build, which
+            should rely on @shopify/scripts-toolchain-as to compile to
+            WebAssembly.
```

Comment:
Depending on the size of the users terminal, the output here might look a little wonky as it'll wrap the lines to display them all. I feel like it would be better to keep this string all on the same line - which probably means we'd have to avoid a heredoc or ignore the long line lint cop 😕 Thats why we usually just use strings and use newline characters wherever we need them. But it could be messy considering this is a larger piece of test.

Though, I like the idea of passing in examples. Probably could be useful in the future too. Maybe it makes sense to add in an `example` argument to `ErrorHandler.display` and create its own entry in messages?

Reply:
Yeah that makes sense, examples is not something that I see frequently, so, for now, I'll just avoid the heredoc, also since in the long term we don't plan to rely on ensuring that the dev must use our toolchain, adding an example entry might only add overhead. I do agree that if we end up adding more examples down the road, adding a dedicated entry is the best.

Reply:
Fixed!